### ISSUE TITLE
Graph QL Collections lack features of regular rest collections (visualizers, packages, copy as curl, etc)

### ISSUE THREAD

**Issue by rtaycher** [a85/POSTMan-Chrome-Extension#13354]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

Graph QL collections lack many of the features of regular collections. If possible it would be preferable for them to be part of regular collections and work the same way. Since that might take a lot of work I'd at least like to seem have the same features. 

Graphql can't specify Authentication or Scripts on Folders. 

Copying as curl(or as python/c#/rust/etc.) is also insanely useful and doesn't seem to be available for GrapQL.

The after response script (named differently from post response in rest collections for some reason) doesn't even support pm.response.code
and doesn't let you import packages with pm.require or set visualizers in with pm.visalizer.set  .
Not sure if it supports await or pm.sendRequest.

The GraphQL examples also lack Try buttons like the rest examples have.

Basically it's nice that you sort of support Graph QL  but until it acts more like a regular postman request/collection it's much less useful then it can/should be. I wouldn't be surprised if some people are just saving rest collections sending GQL to get around these issues. 

### Steps To Reproduce

Make a GraphQL collection and observe lack of features. Try to write a basic post response script

### Operating System

macOS

### Postman Version

11.13.3

### Postman Platform

Postman App

### User Account Type

Signed In User
